src/frontend/window/main_window.rs

```diff
@@ -20,29 +20,50 @@ pub enum AppInput {
     ScreenshotWindowOutput(ScreenshotWindowOutput),
 }
 
+/// The main struct of `flakeshot` which will manage the
+/// - windows and
+/// - tray
+///
+/// of `flakeshot`.
```

Comment:
Avoid enumeration of responsibilities. This is subject to change.

```suggestion
/// The main struct of `flakeshot` which will manage the application lifecycle.
```